src/applications/income-limits/reducers/index.js

```diff
@@ -5,6 +5,24 @@ import {
 } from '../constants';
 
 /* eslint-disable camelcase */
+// const initialState = {
```

Comment:
Note: this is the "standard" case as opposed to the Jonesboro case defined below. Since we are still hard-coding API data, you have to pull locally and flip to one or the other to see the different flows.